packages/docs/static/data/medplumDefinitions/accesspolicy.json

```diff
@@ -289,7 +337,7 @@
       "depth": 2,
       "types": [
         {
-          "datatype": "string"
+          "datatype": "code"
```

Comment:
Nice, good catch 👍

Reply:
Actually, you found lots of good bugs in the structure definitions!  thank you!  🐑

Reply:
I just ran the generator and these changes were emitted; do we need to add something to the release process to ensure all relevant generator scripts get executed?